src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -1,11 +1,12 @@
 using System.Windows.Input;
 using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Controls;
 
 namespace CommunityToolkit.Maui.Views;
 
 /// <inheritdoc cref="IExpander"/>
 [ContentProperty(nameof(Content))]
-public class Expander : StackLayout, IExpander
+public class Expander : ContentView, IExpander
```

Comment:
can we make this a `Grid` instead of a `ContentView`?

Reply:
Unfortunately no, if we make it grid the content is not wrapped (on C# sample page)